posthog/temporal/workflows/squash_person_overrides.py

```diff
@@ -743,16 +789,11 @@ async def run(self, inputs: SquashPersonOverridesInputs):
                 workflow.logger.info("No overrides to delete were found, workflow done")
                 return
 
+            query_inputs.person_overrides_to_delete = persons_to_delete
```

Comment:
I find this hard to read (& other places, where we change the object) - it's not intuitively clear that it's changing it the right way vs before the QueryInputs were explicitly using a new object.